AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/Model/KeyStore.smithy

```diff
@@ -146,6 +146,9 @@ structure GetKeyStoreInfoOutput {
   @javadoc("An identifier for this Key Store.")
   keyStoreId: String,
   @required
+  @javadoc("An identifier to be used in the cache for this Key Store.")
+  keyStoreCacheId: String,
```

Comment:
Blocking: Not a String.
A blob.

Reply:
Yeah I was just experimenting here. I am more concerned about the variable not being able to `UTF8.Encode(config.id.value)` and `uuid bytes` types at the same time.